clear_lambda_storage.py

```diff
@@ -4,7 +4,7 @@
 from __future__ import print_function
 import argparse
 import boto3
-import queue
+import Queue as queue
```

Comment:
Can you please make it safe for both Python 2 and 3?

Reply:
sure, will update it.

Reply:
@ranrib done.